packages/create-vite/template-react-ts/eslint.config.js

```diff
@@ -0,0 +1,32 @@
+import js from '@eslint/js'
+import globals from 'globals'
+import reactHooks from 'eslint-plugin-react-hooks'
+import reactRefresh from 'eslint-plugin-react-refresh'
+import tsParser from '@typescript-eslint/parser'
+import tsPlugin from '@typescript-eslint/eslint-plugin'
+
+export default [
```

Comment:
I think we should do:
```suggestion
import tseslint from 'typescript-eslint';

export default tseslint.config({
```

Reply:
I actually did try to migrate to `typescript-eslint` too, but their docs have rules about spreading them elsewhere, a lot different from our current config, so I didn't want to touch them for now and it could be done as a follow-up, maybe with https://github.com/vitejs/vite/pull/17624

Reply:
The advanced version I kinked above is really close to out config, it's just wrapping and updating import I think

Reply:
done! I've tested that it works as expected